ivy_tests/test_ivy/test_functional/test_core/test_elementwise.py

```diff
@@ -693,12 +689,10 @@ def test_divide(
     input_dtype, x = dtype_and_x
     x1 = np.asarray(x[0], dtype=input_dtype[0])
     x2 = np.asarray(x[1], dtype=input_dtype[1])
-    # ToDo: remove the checks below, and instead handle this during the
-    #  hypothesis data generation
-    if np.any(x2 == 0):
-        return  # don't divide by 0
-    elif np.any(x1 > 9223372036854775807):
-        return  # np.divide converts to signed int so values can't be too large
+    # np.divide converts to signed int so values can't be too large
+    assume(np.all(x1 <= 9223372036854775807))
```

Comment:
is this needed, with the new `0.95` safety factor in the [data generation helpers](https://github.com/unifyai/ivy/blob/591ac37a664ebdf2ca50a5b0751a3a54ee9d5934/ivy_tests/test_ivy/helpers.py#L1470)?